samples/androidNativeActivity/README.md

```diff
@@ -0,0 +1,8 @@
+# Android Native Activity
+ This example shows how to build an Android Native Activity.
+
+The example will render a textured dodecahedron using OpenGL ES library. You can rotate it with your fingers.
+
+To build use `ANDROID_HOME=<your path to android sdk> ../gradlew buildApk`.
+
+Run `adb install build/outputs/apk/androidNativeActivity-arm-debug.apk` to deploy the apk on your android.
```

Comment:
on your android -> on Android device or emulator (note that only ARM-based devices are currently supported).